src/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixer.php

```diff
@@ -42,7 +42,12 @@ final class NativeFunctionTypeDeclarationCasingFixer extends AbstractFixer
      * object   PHP 7.2
```

Comment:
I believe that with each PR adding support for PHP 8.x we should extend the 8.x integration test

Reply:
Integration test is added in #6621 🙂 I think it should be merged first and act as a base for further works.